Documentation/Signoffs/Start Sensor/Detail_Design.md

```diff
@@ -0,0 +1,101 @@
+# Start Sensor System
+
+## Function of the Subsystem
+
+![Alt text](https://github.com/cebttu/CapstoneTeam1/blob/CazBilbrey-StartSensor-SignOff/Documentation/Signoffs/Start%20Sensor/Start%20Block%20Diagram.png)
+Figure 1: Start Sensor Block Diagram
+
+The Start Sensor System serves as the critical initiation mechanism for the robot, effectively kickstarting its traversal along the predefined course. This pivotal system harnesses the capabilities of an RGB color sensor, programmed to recognize the presence of a green LED that signals the beginning of the run. When the sensor is powered via the power system and then detects its specific color, it promptly communicates this information to the Main Controller. 
+
+The Main Controller, acting upon this input, will then activate the Start State, setting the robot in motion to fulfill its intended tasks for the rest of the run. This elegant choreography ensures that the robot remains stationary and inert until the requisite green signal is identified. 
+
+To safeguard against any potential hiccups or sensor malfunctions, a contingency plan is in place. A fail-safe option has been incorporated: a manual button that can be pressed if the sensor fails to perform as expected. These two input sources are connected via an OR Gate, ensuring that the robot can commence its operation seamlessly either when the RGB sensor detects green light or when the manual button is engaged. This redundancy increases the robot's reliability and resilience in real-world applications.
+
+## Constraints and Standards
+
+| Number | Constraint Description | Origin |
+|--------|------------------------|--------|
+| 1. | The RGB Sensor shall detect a spectrum of green light from an LED | Competition Specifications |
+| 2. | The RGB Sensor shall consider at least 4.0V as a HIGH signal and anything lower than 4.0V as a LOW signal | Sensor Specifications |
+| 3. | The RGB Sensor shall detect the light within 3 seconds of light being turned on | Strategy Specifications |
+| 4. | The RGB Sensor shall be located in a height in range of the Green Light LED location | Competition and Sensor Specifications |
+| 5. | The RGB Sensor shall be located in a close in a length range of the Green Light LED location | Competition and Sensor Specifications |
+| 6. | The RGB Sensor shall be located in within a width in range of the Green Light LED location | Competition and Sensor Specifications |
+
+## Buildable Schematic
```

Comment:
Not completely buildable. Pins aren't all labeled. 

Also, it is going to make your design difficult to manage if you use a software that is not typically used for electrical schematics. Kicad is ideal.